include/seqan3/range/container/small_vector.hpp

```diff
@@ -60,6 +60,7 @@ class small_vector
     using const_iterator  = value_type const *;                   //!< The const_iterator type.
     using difference_type = ptrdiff_t;                            //!< The difference_type type.
     using size_type       = detail::min_viable_uint_t<capacity_>; //!< The size_type type.
+
```

Comment:
?

Reply:
doxygen :roll_eyes: 
If there is no space here the size_type of the derived child has no documentation for size_type...

Reply:
![image](https://user-images.githubusercontent.com/2996863/57075032-29679980-6cd5-11e9-8013-5c0553a810bb.png)